misc/python/materialize/mzcompose/services.py

```diff
@@ -95,7 +97,10 @@ def __init__(
                 # HTTP and SQL ports in older versions of Materialize are the same
                 config_ports.pop()
 
-        command_list = []
+        command_list = [
```

Comment:
Item No 4 . Unfortunately adding options to services.py like that prevents mzcompose from starting older-version Mz instances , which is used in the feature-benchmark and possibly in the future in upgrade tests. The better way to do this would be to set those as environment variables, as done for `MZ_PERSIST_BLOB_URL` on line 60 of this file. This way Mz instances that do not support those options will ignore them.

Reply:
Done.